config/proto/targets.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto2";
+
+package hermes.targets
+
+option go_package = "github.com/googleinterns/hermes/targets/proto";
+
+message TargetsDef {
+  enum TargetSystem {
+    GOOGLE_CLOUD_STORAGE = 0;
+    CEPH = 1;
+  }
+
+  enum ConnectionType {
+   HTTP = 0;
+   HTTPS = 1;
+  }
+
+  required TargetSystem target_system = 1;
+  ConnectionType connection_type = 2;
+  ConnectionDetails connection_details = 3;
+  required int64 total_space_allocated = 4;
+}
+
+message ConnectionDetails {
+  string target_url = 1;
+  int port = 2;
+  string api_key = 3;
```

Comment:
Which API? Where can I (the user) get this key from? When will it be needed? Add comments above.

Reply:
Added suggestion:
```suggestion
  
  // API key for the Ceph S3 API (if using Ceph) - irrelevant for GCS.
  // API key can be gotten from your Ceph radosgw instance.
  // Necessary for interacting with the Ceph S3 API.
  string api_key = 3;
```